sdk/data/azappconfig/options.go

```diff
@@ -28,6 +28,8 @@ type DeleteSettingOptions struct {
 
 	// If set, and the configuration setting exists in the configuration store,
 	// delete the setting if the passed-in ETag is the same as the setting's ETag in the configuration store.
+	//
+	// This has IfMatch semantics.
 	OnlyIfUnchanged *azcore.ETag
```

Comment:
We could mark these as deprecated and add the standard names. Might add clutter though. Thoughts?